pages/index.tsx

```diff
@@ -73,7 +73,7 @@ export default function HomePage({
   );
   const [activeFilters, setActiveFilters] = useState<PickerFilter[]>([]);
   const [filtersList, setFiltersList] = useState<PickerFilter[]>(
-    initialState.focuses
+    initialState.focuses.slice(0, 7)
```

Comment:
@taylorkmho looking back at our conversation per last PR (https://github.com/hawaiians/hawaiiansintech/pull/6#discussion_r1002810010), re-engineering the 2 line handling would take a bit more time but thought limiting to 7 choices with the 'View All' control would be a lot better than the current functionality.

Reply:
This works with me! I think the longer term solution would be a little more thoughtful toward how it differentiated with mobile. Let's just say 7 items still gets fairly tall. But this is 200% an improvement from the current behavior.